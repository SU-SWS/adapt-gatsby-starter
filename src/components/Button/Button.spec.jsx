import React from 'react';
import renderer from 'react-test-renderer';
import { Button } from './Button';

describe('Button', () => {
  it('should render a Button', () => {
    const click = jest.fn();
    const component = renderer.create(
      <Button onClick={click}>Click Me</Button>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
