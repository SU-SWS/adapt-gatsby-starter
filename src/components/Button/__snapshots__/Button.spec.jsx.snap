// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render a Button 1`] = `
<button
  onClick={[MockFunction]}
  type="button"
>
  Click Me
</button>
`;
